### ISSUE TITLE
Possible bug when product_version is *-released

### ISSUE THREAD

**Issue by juliogonzalez** [moio/sumaform#428]
As far as I can see at:
https://github.com/moio/sumaform/blob/master/salt/repos/suse_manager_server.sls
https://github.com/moio/sumaform/blob/master/salt/repos/suse_manager_proxy.sls

We have things like:
```
{% if '3.0' in grains['product_version'] %}
```
```
{% if '3.1' in grains['product_version'] %}
```
```
{% if '3.2' in grains['product_version'] %}
```

But according to the ADVANCED_README:
```
## Changing product versions

Some modules have a `product_version` variable that determines the software product version. Specifically:
 * in `suse_manager`, `suse_manager_proxy` etc. `product_version` determines the SUSE Manager product version,
 * in `minion`, `client`, etc. `product_version` determines the SUSE Manager Tools version.

Legal values for released software are:
 * `3.0-released` (latest released Maintenance Update for SUSE Manager 3.0 and Tools)
 * `3.1-released` (latest released Maintenance Update for SUSE Manager 3.1 and Tools)
 * `3.2-released` (latest released alpha/beta/gold master candidate for SUSE Manager 3.2 and Tools)
 * `stable`       (latest released version for Uyuni, from systemsmanagement:Uyuni:Stable)

Legal values for work-in-progress software are:
 * `3.0-nightly` (corresponds to the Build Service project Devel:Galaxy:Manager:3.0)
 * `3.1-nightly` (corresponds to the Build Service project Devel:Galaxy:Manager:3.1)
 * `head` (corresponds to the Build Service project Devel:Galaxy:Manager:Head if OS is SLE12 or systemsmanagement:Uyuni:Master otherwise)
 * `test` (corresponds to the Build Service project Devel:Galaxy:Manager:TEST)
```

**Comment by moio** [comment]
The Jinja `in` operator is called `substring` in other languages. `'3.2' in grains['product_version']` is true for `3.2-released` and `3.2-nightly` and that is intentional.

I fixed the doc.

**Comment by juliogonzalez** [comment]
Yes, of course, I don't know what I was thinking about yesterday. For some reason I saw the `=` operator,  not `in`
